### ISSUE TITLE
Dynamic line thickness for Spectral plots

### ISSUE THREAD

**Issue by borondics** [Quasars/orange-spectroscopy#382]
When displaying a few, i.e. <10 curves, Spectra should draw with 2pt line thickness for better visualization.

**Comment by stuart-cls** [comment]
@markotoplak wasn't there some issue that related line thickness to plotting speed a while ago?

**Comment by markotoplak** [comment]
The issue still exists, but we could handle some low number of curves as suggested in this PR.

**Comment by borondics** [comment]
Yes, for many curves we can use thin lines, there will be many anyway, which will make them visible.
